README.md

```diff
@@ -618,19 +622,41 @@ P.S. Look at [testify's issue](https://github.com/stretchr/testify/issues/772),
 
 ```go
 ❌
-assert.Equal(t, 3, len(arr))
-assert.EqualValues(t, 3, len(arr))
-assert.Exactly(t, 3, len(arr))
-assert.True(t, len(arr) == 3)
+assert.Equal(t, 42, len(arr))
+assert.Equal(t, len(arr), 42)
+assert.EqualValues(t, 42, len(arr))
+assert.EqualValues(t, len(arr), 42)
+assert.Exactly(t, 42, len(arr))
+assert.Exactly(t, len(arr), 42)
+assert.True(t, 42 == len(arr))
+assert.True(t, len(arr) == 42)
+
+assert.Equal(t, value, len(arr))
+assert.EqualValues(t, value, len(arr))
+assert.Exactly(t, value, len(arr))
+assert.True(t, len(arr) == value)
+
+assert.Equal(t, len(expArr), len(arr))
+assert.EqualValues(t, len(expArr), len(arr))
+assert.Exactly(t, len(expArr), len(arr))
+assert.True(t, len(arr) == len(expArr))
 
 ✅
-assert.Len(t, arr, 3)
+assert.Len(t, arr, 42)
+assert.Len(t, arr, value)
+assert.Len(t, arr, len(expArr))
 ```
 
 **Autofix**: true. <br>
 **Enabled by default**: true. <br>
 **Reason**: More appropriate `testify` API with clearer failure message.
 
+> [!CAUTION]
+> The checker ignores assertions in which length checking is not a priority, e.g.
```

Comment:
Maybe a link to the issue where the reason are provided could help people to understand it's normal, and it would be a bad thing to change